rune/libenclave/attestation/sgx/ias/ias.go

```diff
@@ -354,8 +369,13 @@ func checkVerificationReport(resp *http.Response, quote []byte, nonce string) (*
 
 	if report.IsvEnclaveQuoteStatus == "GROUP_OUT_OF_DATE" ||
 		report.IsvEnclaveQuoteStatus == "CONFIGURATION_NEEDED" {
-		if resp.Header.Get("Advisory-Ids") == "" ||
-			resp.Header.Get("Advisory-Url") == "" {
+		if report.Version == 3 {
+			if resp.Header.Get("Advisory-Ids") == "" || resp.Header.Get("Advisory-Url") == "" {
+				return status, fmt.Errorf("Advisory-Ids or Advisory-Url is not present in response header")
+			}
+		}
```

Comment:
Change to `} else if  report.Version == 4 && (report.AdvisoryIds == "" || report.AdvisoryUrl == nil) {`